Update to make compatible with Reaction v1.11.0

Is the project at all runnable under 1.11.0?

The error I get (upon running `export SKIP_FIXTURES=1 && reaction`) is:

```
Error: Cannot find module '../../../../../core/collections'
    at makeMissingError (packages/modules-runtime.js:231:12)
    at require (packages/modules-runtime.js:241:19)
    at tags.js (imports/plugins/custom/reaction-swag-shop/lib/collections/schemas/tags.js:1:29)
    at fileEvaluate (packages/modules-runtime.js:343:9)
    at require (packages/modules-runtime.js:238:16)
    at index.js (imports/plugins/custom/reaction-swag-shop/lib/collections/schemas/index.js:1:14)
    at fileEvaluate (packages/modules-runtime.js:343:9)
    at require (packages/modules-runtime.js:238:16)
    at index.js (imports/plugins/custom/reaction-swag-shop/server/index.js:1:14)
    at fileEvaluate (packages/modules-runtime.js:343:9)
    at require (packages/modules-runtime.js:238:16)
    at plugins.js (server/plugins.js:1:3419)
    at fileEvaluate (packages/modules-runtime.js:343:9)
    at require (packages/modules-runtime.js:238:16)
    at /Users/tsenkov/repos/example-store/.meteor/local/build/programs/server/app/app.js:110231:1
    at /Users/tsenkov/repos/example-store/.meteor/local/build/programs/server/boot.js:411:36
    at Array.forEach (<anonymous>)
    at /Users/tsenkov/repos/example-store/.meteor/local/build/programs/server/boot.js:220:19
    at /Users/tsenkov/repos/example-store/.meteor/local/build/programs/server/boot.js:471:5
    at Function.run (/Users/tsenkov/repos/example-store/.meteor/local/build/programs/server/profile.js:510:12)
    at /Users/tsenkov/repos/example-store/.meteor/local/build/programs/server/boot.js:470:11
```

this is the whole error in terminal
`reaction_1   | /home/node/.meteor/packages/meteor-tool/.1.6.1_1.164hzk5.nrx7++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
reaction_1   | 						throw(ex);
reaction_1   | 						^
reaction_1   | 
reaction_1   | Error: Cannot find module '../../../../../core/collections'
reaction_1   |     at makeMissingError (packages/modules-runtime.js:231:12)
reaction_1   |     at require (packages/modules-runtime.js:241:19)
reaction_1   |     at tags.js (imports/plugins/custom/reaction-swag-shop/lib/collections/schemas/tags.js:1:29)
reaction_1   |     at fileEvaluate (packages/modules-runtime.js:343:9)
reaction_1   |     at require (packages/modules-runtime.js:238:16)
reaction_1   |     at index.js (imports/plugins/custom/reaction-swag-shop/lib/collections/schemas/index.js:1:14)
reaction_1   |     at fileEvaluate (packages/modules-runtime.js:343:9)
reaction_1   |     at require (packages/modules-runtime.js:238:16)
reaction_1   |     at index.js (imports/plugins/custom/reaction-swag-shop/server/index.js:1:14)
reaction_1   |     at fileEvaluate (packages/modules-runtime.js:343:9)
reaction_1   |     at require (packages/modules-runtime.js:238:16)
reaction_1   |     at plugins.js (server/plugins.js:1:3419)
reaction_1   |     at fileEvaluate (packages/modules-runtime.js:343:9)
reaction_1   |     at require (packages/modules-runtime.js:238:16)
reaction_1   |     at /opt/reaction/src/.meteor/local/build/programs/server/app/app.js:110393:1
reaction_1   |     at /opt/reaction/src/.meteor/local/build/programs/server/boot.js:411:36
reaction_1   |     at Array.forEach (<anonymous>)
reaction_1   |     at /opt/reaction/src/.meteor/local/build/programs/server/boot.js:220:19
reaction_1   |     at /opt/reaction/src/.meteor/local/build/programs/server/boot.js:471:5
reaction_1   |     at Function.run (/opt/reaction/src/.meteor/local/build/programs/server/profile.js:510:12)
reaction_1   |     at /opt/reaction/src/.meteor/local/build/programs/server/boot.js:470:11
reaction_1   | => Exited with code: 1`
thanks in advanced

--- a/client/components/footer.js
+++ b/client/components/footer.js
@@ -3,7 +3,7 @@ import { Meteor } from "meteor/meteor";
 import SimpleSchema from "simpl-schema";
 import Alert from "sweetalert2";
 import { i18next } from "/client/api";
-import { Validation } from "@reactioncommerce/reaction-collections";
+import { Validation } from "@reactioncommerce/schemas";
 import { registerComponent, Components } from "@reactioncommerce/reaction-components";
 import { Card, CardHeader, CardBody } from "/imports/plugins/core/ui/client/components";
 

--- a/client/components/product-variant/customer/productGrid.js
+++ b/client/components/product-variant/customer/productGrid.js
@@ -129,12 +129,6 @@ class ProductGrid extends ProductGridCore {
     return chunks;
   }
 
-  shopAllLabel() {
-    return (
-      <span><Components.Translation defaultValue={"Shop all products"} i18nKey={"shopAllProducts"} />&nbsp;<i className="fa fa-long-arrow-right" /></span>
-    );
-  }
-
   renderCategories() {
     return (
       <div className={"categories row"}>
@@ -143,7 +137,10 @@ class ProductGrid extends ProductGridCore {
             <div className={"btn-essentials"}>
               <Components.Button
                 className={"btn-blue"}
-                label={this.shopAllLabel()}
+                label={"Shop all products"}
+                i18nKeyLabel={"shopAllProducts"}
+                icon={"fa fa-long-arrow-right"}
+                iconAfter
                 bezelStyle={"solid"}
                 primary={false}
                 type="button"

--- a/client/styles/classes/landing.less
+++ b/client/styles/classes/landing.less
@@ -89,6 +89,10 @@
   bottom: 10%;
   text-align: center;
   width: 100%;
+
+  > i {
+    margin-left: 8px;
+  }
 }
 /* EOF category tiles */
 

--- a/lib/collections/schemas/swagProduct.js
+++ b/lib/collections/schemas/swagProduct.js
@@ -1,8 +1,7 @@
 import { Meteor } from "meteor/meteor";
-import { getSchemas } from "@reactioncommerce/reaction-collections";
+import Schemas from "@reactioncommerce/schemas";
 import { Products } from "/lib/collections/index";
 
-const Schemas = getSchemas();
 const ExtendedSchema = Schemas.Product.extend({
   featuredProductLabel: {
     optional: true,

--- a/lib/collections/schemas/tags.js
+++ b/lib/collections/schemas/tags.js
@@ -1,7 +1,6 @@
-import { getSchemas } from "@reactioncommerce/reaction-collections";
+import Schemas from "@reactioncommerce/schemas";
 import { Tags } from "/lib/collections";
 
-const Schemas = getSchemas();
 const TagSchema = Schemas.Tag.extend({
   catTileImageUrl: {
     type: String,

--- a/server/publications/collections/products.js
+++ b/server/publications/collections/products.js
@@ -1,13 +1,11 @@
 import _ from "lodash";
 import { Meteor } from "meteor/meteor";
 import { check, Match } from "meteor/check";
+import Schemas from "@reactioncommerce/schemas";
 import { Shops, Tags, Catalog } from "/lib/collections";
 import { Logger } from "/server/api";
-import { getSchemas } from "@reactioncommerce/reaction-collections";
-
 
 // Validate the subscription filter against our extended filter schema.
-const Schemas = getSchemas();
 const { filters } = Schemas;
 
 function filterProducts(productFilters) {

--- a/server/publications/collections/schemas/filters.js
+++ b/server/publications/collections/schemas/filters.js
@@ -1,6 +1,5 @@
-import { getSchemas } from "@reactioncommerce/reaction-collections";
+import Schemas from "@reactioncommerce/schemas";
 
-const Schemas = getSchemas();
 Schemas.filters.extend({
   featuredProductLabel: {
     type: String,

